api/apps/v1alpha1/nemo_entitystore_types.go

```diff
@@ -510,6 +506,25 @@ func (n *NemoEntitystore) GetDeploymentParams() *rendertypes.DeploymentParams {
 
 	// Set runtime class
 	params.RuntimeClassName = n.GetRuntimeClass()
+
+	// Extract ports from spec and update rendering params
+	if len(n.GetServicePorts()) > 0 {
+		var containerPorts []corev1.ContainerPort
+		for _, svcPort := range n.GetServicePorts() {
```

Comment:
common nit across files:
```suggestion
        svcPorts := n.GetServicePorts()
	if len(svcPorts) > 0 {
		var containerPorts []corev1.ContainerPort
		for _, svcPort := range svcPorts {
```

Reply:
done